runtime/src/bank/pyth_accumulator.rs

```diff
@@ -367,3 +382,32 @@ pub fn update_v2(bank: &Bank) -> std::result::Result<(), AccumulatorUpdateErrorV
 
     update_v1(bank, &v2_messages, any_v1_aggregations)
 }
+
+pub fn compute_publisher_stake_caps(
+    bank : &Bank,
+    accounts : &Vec<(Pubkey, AccountSharedData)>, 
+    timestamp : i64,
+    messages : &mut Vec<Vec<u8>>,
+    is_active : bool
+) {
+    let parameters: StakeCapParameters = {
+        let data = bank
+        .get_account_with_fixed_root(&STAKE_CAPS_PARAMETERS_ADDR)
+        .unwrap_or_default();
+    let data = data.data();
+    solana_sdk::borsh::try_from_slice_unchecked(data)
+        .unwrap_or(StakeCapParameters { _discriminator: 0, _current_authority : [0u8;32], z: 1, m : 1_000_000_000 })
+};
+
+    info!("Computing publisher stake caps with m : {} and z : {}", parameters.m, parameters.z);
+    let account_datas : Vec<&[u8]> = accounts
+        .iter()
+        .map(|(_, account)| account.data().borrow())
+        .collect();
+    let message = pyth_oracle::validator::compute_publisher_stake_caps(account_datas, timestamp, parameters.m, parameters.z);
```

Comment:
I want to compute this if `is_active` is false so that if there's a bug or it crashes during the computation, we realize when validators update individually and not when the flag gets activated